kv/mem_buffer_test.go

```diff
@@ -42,14 +42,12 @@ type testKVSuite struct {
 }
 
 func (s *testKVSuite) SetUpSuite(c *C) {
-	s.bs = make([]MemBuffer, 2)
-	s.bs[0] = NewRBTreeBuffer()
-	s.bs[1] = NewMemDbBuffer()
+	s.bs = make([]MemBuffer, 1)
```

Comment:
To keep the line of changes as minimum as possible.
Make be we need to add another MemBuffer implementation later.